paddle/inference/io.cc

```diff
@@ -21,6 +21,18 @@ limitations under the License. */
 namespace paddle {
 namespace inference {
 
+void ReadProgramDescFromFile(const std::string& model_filename,
+                             std::string& program_desc_str) {
```

Comment:
I think this function is a common function to read binary to string. If there is no other use for this reading function, how about change it to
```cpp
std::unique_ptr<framework::ProgramDesc> ReadProgram(const std::string& filename)
```

But we can change this in next PR.

Reply:
I think this function will also be useful for "loading-from-buffer" case. I have changed the name to `ReadBinaryFile`.